test_scripts/RC/ButtonPress/011_generic_error_if_read_only_response_come_from_hmi.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/9
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/button_press_emulation.md
```

Comment:
actual link is https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/RC/button_press_emulation.md